command/unixOSCommand.go

```diff
@@ -0,0 +1,85 @@
+package command
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/DataDog/test-infra-definitions/common/utils"
+	"github.com/alessio/shellescape"
+	"github.com/pulumi/pulumi-command/sdk/go/command/remote"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+const (
+	linuxTempDir = "/tmp"
+)
+
+var _ osCommand = (*unixOSCommand)(nil)
+
+type unixOSCommand struct{}
+
+func newUnixOSCommand() osCommand {
+	return unixOSCommand{}
+}
+
+func (unixOSCommand) CreateDirectory(
+	runner *Runner,
+	name string,
+	remotePath pulumi.StringInput,
+	useSudo bool,
+	opts ...pulumi.ResourceOption) (*remote.Command, error) {
+	return createDirectory(
+		runner,
+		name,
+		"mkdir -p %s",
+		"rm -rf %s",
+		remotePath,
+		useSudo,
+		opts...)
+}
+
+func (unixOSCommand) CopyInlineFile(
+	runner *Runner,
+	name string,
+	fileContent pulumi.StringInput,
+	remotePath string,
+	useSudo bool,
+	append bool,
+	opts ...pulumi.ResourceOption) (*remote.Command, error) {
+	var createCmd pulumi.StringInput
+	if append {
+		createCmd = utils.AppendStringCommand(remotePath, useSudo)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Are append and write string command shared across windows and unix ? I see they are in `utils`

Reply:
No there are not shared. Please note that `AppendStringCommand` and `WriteStringCommand` are removed in https://github.com/DataDog/test-infra-definitions/pull/67